okhttp/src/main/java/io/grpc/okhttp/OkHttpClientTransport.java

```diff
@@ -1014,9 +1019,17 @@ public void ackSettings() {
 
     @Override
     public void goAway(int lastGoodStreamId, ErrorCode errorCode, ByteString debugData) {
+      if (errorCode == ErrorCode.ENHANCE_YOUR_CALM) {
+        String data = debugData.utf8();
+        log.log(Level.WARNING, String.format(
+            "%s: Received GOAWAY with ENHANCE_YOUR_CALM. Debug data: %s", this, data));
```

Comment:
This is the difference between `MessageFormat` and `String.format()`. logcat in Android generally doesn't handle the replacements, so we've worked around it by manually making sure the replacement happens before passing it to the logging infrastructure.

I'm pretty sure we could have used MessageFormat, but I didn't do that and I'm just being consistent with the rest of the code.